README.md

```diff
@@ -42,13 +42,20 @@ img.data
 ```
 
 ### Reading from AWS S3
-To read from private S3 buckets or public buckets using `s3://` paths, [credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) must be configured. Public buckets can be accessed without credentials by using the `https://` path.
+To read from private S3 buckets, [credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) must be configured. Public buckets can be accessed without credentials.
 ```python
 from bioio import BioImage
 path = "https://allencell.s3.amazonaws.com/aics/nuc-morph-dataset/hipsc_fov_nuclei_timelapse_dataset/hipsc_fov_nuclei_timelapse_data_used_for_analysis/baseline_colonies_fov_timelapse_dataset/20200323_09_small/raw.ome.zarr"
 image = BioImage(path)
 print(image.get_image_dask_data())
 ```
+If using an `s3://` path to access a public S3 bucket, the `BioImage` constructor must be given a dictionary with `anon: True` in the `fs_kwargs` argument.
```

Comment:
this is good and valuable documentation but slightly unfortunate that it lives here in the reader's README.  It would be great if somewhere in the bioio docs we said something like "be sure to check the readmes or documentation for individual readers, for their specific parameters." It might already be there...  This is just an occupational hazard of having readers in separate packages.

Reply:
This particular piece of documentation may apply to all/many readers.

Reply:
Filed an issue for it https://github.com/bioio-devs/bioio/issues/60